powerquery-docs/connectors/intersystems-iris.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Intersystems IRIS
+description: Provides basic details for the Intersystems IRIS connector for Power BI.
+author: Julie Bolinsky
+ms.topic: conceptual
+ms.date: 5/3/2024
+ms.author: jbolinsk
```

Comment:
The ms.author metadata must be a Microsoft employee. You can use my Microsoft alias for this metadata - dougklo